appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleEvent.kt

```diff
@@ -25,9 +31,11 @@ internal sealed class ExperienceLifecycleEvent(
         override val experience: Experience,
         val stepIndex: Int,
         val interactionType: InteractionType,
+        val interactionProperties: HashMap<String, Any> = hashMapOf()
     ) : ExperienceLifecycleEvent(experience, AnalyticsEvent.ExperienceStepInteraction) {
+
         enum class InteractionType {
-            FORM_SUBMITTED
+            FORM_SUBMITTED, BUTTON_TAPPED, BUTTON_LONG_PRESSED
```

Comment:
I was initially implementing both in a separate spot but I figured that they are similar to FORM_SUBMITTED and it should reside in here (let me know if thats how you imagined that would work)